src/NuGet.Core/NuGet.Common/PublicAPI/net45/PublicAPI.Unshipped.txt

```diff
@@ -1,10 +0,0 @@
-NuGet.Common.IPackLogMessage.Framework.get -> NuGet.Frameworks.NuGetFramework
-NuGet.Common.IPackLogMessage.Framework.set -> void
-NuGet.Common.IPackLogMessage.LibraryId.get -> string
-NuGet.Common.IPackLogMessage.LibraryId.set -> void
-NuGet.Common.NuGetLogCode.NU5042 = 5042 -> NuGet.Common.NuGetLogCode
-NuGet.Common.PackagingLogMessage.Framework.get -> NuGet.Frameworks.NuGetFramework
-NuGet.Common.PackagingLogMessage.Framework.set -> void
-NuGet.Common.PackagingLogMessage.LibraryId.get -> string
-NuGet.Common.PackagingLogMessage.LibraryId.set -> void
-static NuGet.Common.PackagingLogMessage.CreateWarning(string message, NuGet.Common.NuGetLogCode code, string libraryId, NuGet.Frameworks.NuGetFramework framework) -> NuGet.Common.PackagingLogMessage
```

Comment:
The `Shipped` file has only one addition, but the `Unshipped` file has 10 subtractions. Plus CI is failing, probably because of the APIs that are not missing from either file.

Reply:
Fixed in the latest commit.